vars/pipelineEthereumEvents.groovy

```diff
@@ -0,0 +1,43 @@
+// https://github.com/vegaprotocol/jenkins-shared-library/issues/373
+def call() {
+    settings = [
+        sandbox: [
+            party: '1a94865a473e547c3e284f266af627856680cadf29fb52aecb3bf72ffff447ad',
+            erc20: '0x51d9Dbe9a724C6a8383016FaD566e55c95359D36',
+        ]
+    ]
+    pipeline {
+        agent any
+        options {
+            timestamps()
+            ansiColor('xterm')
+            lock(resource: env.NET_NAME)
+        }
+        stages {
+            stage('Checkout') {
+                steps {
+                    gitClone(
+                        vegaUrl: 'devopstools',
+                        directory: 'devopstools',
+                        branch: params.DEVOPSTOOLS_BRANCH,
+                    )
+                    dir ('devopstools') {
+                        sh 'go mod download'
+                    }
+                }
+            }
+            stage('Send ethereum events') {
+                steps {
+                    withDevopstools(
+                        command: """topup parties \
+                            --parties ${settings[env.NET_NAME].party} \
+                            --amount 0.0001 \
+                            --erc20-token-address ${settings[env.NET_NAME].erc20} \
+                            --repeat 10
```

Comment:
Would it be possible to take the repeat value from pipeline parameters?
(and name it there something like `number of events`)